its/src/test/kotlin/org/sonarlint/intellij/its/tests/FocusOnNewCodeTest.kt

```diff
@@ -0,0 +1,198 @@
+            "No older Security Hotspots"
+        )
+
+        verifyTaintTabContainsMessages(
+            this,
+            "Found 1 new issue in 1 file  (unsupported new code definition)",
+            "FileWithSink.java",
+            "Change this code to not construct SQL queries directly from user-controlled data.",
+            "No older issues"
+        )
+
+        verifyHotspotTabContainsMessages(
+            this,
+            "Found 1 new Security Hotspot in 1 file  (unsupported new code definition)",
+            "No older Security Hotspots"
+        )
+
+        verifyIssueTreeContainsMessages(
+            this,
+            "Found 2 new issues in 1 file  (unsupported new code definition)",
+            "No older issues"
+        )
+    }
+
+    private fun bindProjectFromPanel() {
+        with(remoteRobot) {
+            idea {
+                toolWindow("SonarLint") {
+                    ensureOpen()
+                    tab("Taint Vulnerabilities") { select() }
+                    content("TaintVulnerabilitiesPanel") {
+                        findText("Configure Binding").click()
+                    }
+                }
+
+                ProjectBindingUtils.bindProjectToSonarQube(
+                    ORCHESTRATOR.server.url,
+                    token,
+                    TAINT_VULNERABILITY_PROJECT_KEY
+                )
+            }
+        }
+    }
+
+    private fun verifyTaintTabContainsMessages(remoteRobot: RemoteRobot, vararg expectedMessages: String) {
+        with(remoteRobot) {
+            idea {
+                toolWindow("SonarLint") {
+                    ensureOpen()
+                    tabTitleContains("Taint Vulnerabilities") { select() }
+                    content("TaintVulnerabilitiesPanel") {
+                        expectedMessages.forEach {
+                            println(data)
+                            Assertions.assertThat(hasText(it)).isTrue()
+                        }
+                    }
+                }
+            }
+        }
+    }
+
+    private fun verifyHotspotTabContainsMessages(remoteRobot: RemoteRobot, vararg expectedMessages: String) {
+        with(remoteRobot) {
+            idea {
+                toolWindow("SonarLint") {
+                    ensureOpen()
+                    tabTitleContains("Security Hotspots") { select() }
+                    content("SecurityHotspotsPanel") {
+                        expectedMessages.forEach { Assertions.assertThat(hasText(it)).isTrue() }
+                    }
+                }
+            }
+        }
+    }
+
+    private fun verifyReportTabContainsMessages(remoteRobot: RemoteRobot, vararg expectedMessages: String) {
+        with(remoteRobot) {
+            idea {
+                toolWindow("SonarLint") {
+                    findCard("PsiAwareTextEditorComponent").rightClick()
+                    actionMenuItem("Analyze with SonarLint") {
```

Comment:
I think it would make sense to have a fixture for this, it's maybe a bit too technical. Ideally, we could have a fixture to launch an analysis and it could be reused for other tests when needed